apps/meteor/client/views/marketplace/AppDetailsPage/tabs/AppSecurity.tsx

```diff
@@ -13,15 +13,15 @@ type AppSecurityProps = {
 	privacyLink?: string;
 };
 
-const AppSecurity: FC<AppSecurityProps> = ({ privacyPolicySummary, appPermissions, tosLink, privacyLink }) => {
+const AppSecurity = ({ privacyPolicySummary, appPermissions, tosLink, privacyLink }: AppSecurityProps): ReactElement => {
 	const t = useTranslation();
 
 	return (
-		<Box maxWidth='x640' w='full' marginInline='auto'>
+		<Box maxWidth='x640' w='full' marginInline='auto' color='default'>
 			<Box display='flex' flexDirection='column'>
 				<Margins block='x17'>
 					<Box is='section'>
-						<Box fontScale='h4' mbe='x8'>
+						<Box fontScale='h4' mbe='x8' color='titles-labels'>
```

Comment:
clear necessity of some component holding all the styles

Reply:
Probably fixed